### PR TITLE
Scrub some strings in the test files

### PR DESCRIPTION
**Pull request by t8y8**
Removing the 'tsi.lan' suffix that's internal to Tableau and not really relevant for general usage

### CODE REVIEW

**Comment by t8y8** [comment]
Ping (I'm just OCD and want to clear my open PRs)

/cc @RussTheAerialist

**Comment by RussTheAerialist** [comment]
I'm oof today because my dog is having a surgery. I'll take a look tomorrow

File: test/bvt.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def test_can_extract_federated_connections(self):
         connections = parser.get_connections()
         self.assertIsInstance(connections, list)
         self.assertIsInstance(connections[0], Connection)
-        self.assertEqual(connections[0].dbname, 'testv1')
+        self.assertEqual(connections[0].dbname, 'TestV1')
```
Comment (LGraber) [inline test/bvt.py]: How did this pass before?

**Comment by LGraber** [comment]
:rocket:

File: test/bvt.py (hunk 1 above)
Comment (t8y8) [inline test/bvt.py]: The test asset had it lowercase too, my OCD said 'no' and I updated it to match the others

Comment (LGraber) [inline test/bvt.py]: That is OOOCCCDDD

**Pull request merged**